man/systemd-analyze.xml

```diff
@@ -243,10 +244,12 @@ multi-user.target @47.820s
     </refsect2>
 
     <refsect2>
-      <title><command>systemd-analyze dump</command></title>
+      <title><command>systemd-analyze dump [<replaceable>pattern</replaceable>…]</command></title>
 
-      <para>This command outputs a (usually very long) human-readable serialization of the complete server
-      state. Its format is subject to change without notice and should not be parsed by applications.</para>
+      <para>Without any parameter, this command outputs a (usually very long) human-readable serialization of
+      the complete server state. Optional pattern globbing style specifications may be passed at the end and
```

Comment:
"Optional pattern globbing style specifications may be passed at the end and in this case the command output will be limited to the units whose names match the passed patterns" → "Optional glob patterns may be specified, causing the output to be limited to units whose names match one of the patterns."

"Its format" → "The output format". (Now that there's much more text, it's not clear if this pertains to the patterns or the output.)